### DISCUSSION TITLE
转换postman脚本异常，提示error="unexpected JSON format"

### DISCUSSION THREAD

**Discussion by yzbf-lin** [httprunner/httprunner#1529] [2 - 答疑（Q&A）]
![image](https://user-images.githubusercontent.com/72718331/205846430-12fcbf14-e691-4ffb-ae9d-d2fe5ba2ea18.png)
转换postman脚本异常，提示error="unexpected JSON format"

**Comment by 2504973175** [comment]
一样的问题，哭了、。。

**Reply by EironnWalker** [reply]
> ![image](https://user-images.githubusercontent.com/72718331/205846430-12fcbf14-e691-4ffb-ae9d-d2fe5ba2ea18.png) 转换postman脚本异常，提示error="unexpected JSON format"

应该是swagger写的有问题导致的，写一个最简单的，然后一点点排查写法吧

**Comment by xuyue1322** [comment]
![image](https://github.com/httprunner/httprunner/assets/102839265/23ebf6c2-f842-4e93-bc5d-e3fe60c7d108)
我转换的时候提示这个错误不知道什么情况

**Reply by hzde0128** [reply]
加上--from-postman参数试试
